### PR TITLE
Implement parameter validation for input_fields

### PR DESCRIPTION
**Pull request by lucassz**
Implement parameter validation for input_fields, both for TaxBrain and for the partial equilibrium model.

### CODE REVIEW

**Comment by hdoupe** [comment]
@lucassz This looks good. Can you add a test demonstrating the validation behavior? There aren't any specific tests for the forms. You can create a new file `taxbrain/tests/test_forms.py` with the new tests. A test with OK parameters and a test with unknown parameters should be sufficient. 

You can look to `webapp/apps/test_assets/test_models.py` and `webapp/apps/taxbrain/tests/test_models.py` for some relevant examples.

**Comment by lucassz** [comment]
@hdoupe Done. It seemed more relevant to add it to `test_views.py` since there was similar code there already.

File: webapp/apps/taxbrain/tests/test_views.py [hunk 1]
```diff
@@ -57,6 +57,20 @@ def test_taxbrain_post(self, data_source):
         check_posted_params(result['tb_dropq_compute'], truth_mods,
                             str(START_YEAR), data_source=data_source)
 
+    @pytest.mark.xfail
```
Comment (hdoupe) [inline webapp/apps/taxbrain/tests/test_views.py]: @lucassz This shouldn't be xfail-ed. That means that `py.test` will ignore it if the test fails.

Comment (lucassz) [reply webapp/apps/taxbrain/tests/test_views.py]: @hdoupe Thank you. I think I had understood the purpose of `xfail`.

**Comment by hdoupe** [comment]
@lucassz This looks great. Thanks for taking care of this and for finding a much simpler way to solve the problem.

**Comment by hdoupe** [comment]
@lucassz This should be good to go once you knock out the merge conflicts.

**Comment by lucassz** [comment]
@hdoupe Happy to do it. Should be good to go now.

**Comment by hdoupe** [comment]
Great, thanks @lucassz

**Pull request merged**